src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Agreed I'm of the belief that polyfills should be included by the implementor too actually. I'm cool with just going ahead with this PR closing to help out @AgrimPrasad, but maybe leave a comment as to why we check the existence of that function?